backend/auth/handlers_status_test.go

```diff
@@ -19,10 +19,10 @@ func TestStatus(t *testing.T) {
 	AddAuthenticationService(router, mockJWTSecret)
 
 	testCases := []struct {
-		name         string
 		dbAdapter    dbadapter.DBInterface
-		expectedCode int
+		name         string
 		expectedBody string
+		expectedCode int
```

Comment:
What how this re-arrangement helps?

Reply:
It helps to reduce/optimize memory footprint